packages/badge/style.ts

```diff
@@ -1,56 +1,54 @@
 import { css } from "emotion";
 import {
   borderRadiusSmall,
-  green,
-  greyDark,
   greyLight,
-  purple,
-  red,
-  white,
-  yellow
+  themeSuccess,
+  themeWarning,
+  themeError,
+  themeBrandPrimary,
+  themeBgPrimary,
+  themeTextColorPrimary,
+  themeTextColorPrimaryInverted,
+  themeBorder
 } from "../design-tokens/build/js/designTokens";
+import { pickReadableTextColor } from "../shared/styles/color";
+import getCSSVarValue from "../utilities/components/getCSSVarValue";
 
-const badgeAppearance = {
-  default: css`
-    background-color: ${greyLight};
-    border-color: ${greyLight};
-    color: ${greyDark};
-  `,
-  success: css`
-    background-color: ${green};
-    border-color: ${green};
-    color: ${white};
-  `,
-  primary: css`
-    background-color: ${purple};
-    border-color: ${purple};
-    color: ${white};
-  `,
-  warning: css`
-    background-color: ${yellow};
-    border-color: ${yellow};
-    color: ${white};
-  `,
-  danger: css`
-    background-color: ${red};
-    border-color: ${red};
-    color: ${white};
-  `,
-  outline: css`
-    background-color: ${white};
-    border-color: ${greyLight};
-    color: ${greyDark};
-  `,
-  "outline-primary": css`
-    background-color: ${white};
-    border-color: ${purple};
-    color: ${greyDark};
-  `
+const badgeAppearanceStyle = (color, isOutlined?: boolean) => {
+  const bgColor = isOutlined ? getCSSVarValue(themeBgPrimary) : color;
+  return css`
+    background-color: ${bgColor};
+    border-color: ${color};
+    color: ${pickReadableTextColor(
+      bgColor,
+      getCSSVarValue(themeTextColorPrimary),
+      getCSSVarValue(themeTextColorPrimaryInverted)
+    )};
+  `;
+};
+
+const badgeAppearance = appearance => {
+  switch (appearance) {
+    case "default":
+      return badgeAppearanceStyle(greyLight);
```

Comment:
Does this mean that default appearance is not themeable?

Reply:
Wooops, this should have been `themeBgNeutral`. Good catch!